### PR TITLE
Release 2.4.0

### PR DESCRIPTION
**Pull request by d4straub**
Notable changes:
- Custom DADA2 reference taxonomy database are supported
- Illumina NovaSeq data can be correctly processed
- ASV length filter is available for stricter QC
- Species classification is now reported in columns "Species" using DADA2's assignTaxonomy function or "Species_exact" using DADA2's addSpecies function, the latter only assigns exact sequence matches.

For detailed changes, please see CHANGELOG.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -4,7 +4,7 @@
     nf-core/ampliseq
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     Github : https://github.com/nf-core/ampliseq
-    Website: https://nf-co.re/ampliseq
+Website: https://nf-co.re/ampliseq
```
Comment (erikrikarddaniel) [inline main.nf:7]: Looks a bit out of line.

**Review by erikrikarddaniel — APPROVED** [review]
Tiny formatting question, otherwise great! Nice work!

File: main.nf (hunk 1 above)
Comment (d4straub) [reply main.nf:7]: I agree, thats the template update's fault, will change.

File: modules/local/vsearch_usearch_global.nf [hunk 2]
```diff
@@ -0,0 +1,33 @@
+process VSEARCH_USEARCH_GLOBAL {
```
Comment (jtangrot) [inline modules/local/vsearch_usearch_global.nf:1]: This process is not used and the file can be removed, but on the other hand it doesn't do any harm either. I must have forgotten to remove it when I made an nf-core module instead.

**Review by jtangrot — APPROVED** [review]
Looks good as far as I can tell! Good work!

File: modules/local/vsearch_usearch_global.nf (hunk 2 above)
Comment (d4straub) [reply modules/local/vsearch_usearch_global.nf:1]: Thanks! Will remove it!

**Comment by d4straub** [comment]
Thanks! Just waiting for the tests to pass...

**Pull request merged**